### PR TITLE
Rewrite X MCP guide for auto-credits and connector-only auth

### PR DESCRIPTION
**Pull request by vaulpannx**
## Summary
- Developer accounts are auto-created and auto-credited. On connect, agents confirm X tools exist, fetch `get_usage_credits`, congratulate without a gift dollar amount, and always state remaining `total_balance` (including \$0.00). Starter amounts by Cursor plan only if the user asks how much they received.
- Connected-but-no-tools (`user-X-*` not found, `tools=0`) is account-not-ready: reinstall the X connection, then console.x.com developer account / Default Project / App. 503s are an X outage, not the user's fault. Never buy-credits copy before a successful credits read.
- Edge cases: do not sign the user into X via GrokBot's browser, and do not ask for Bearer tokens — X plugin Connect only.

## Test plan
- [ ] Connect X with tools present and `free_balance` > 0: congrats line has no gift amount; remaining balance is stated; 2–3 ideas match the budget row.
- [ ] Connect with `total_balance` ~\$0: capabilities + **You have \$0.00 in credits** + free lookups + console.x.com.
- [ ] Connected with `tools=0` / `user-X-*` not found: account-not-ready copy (reinstall, then console project/app). Not a paywall.
- [ ] `get_usage_credits` 503 while `get_users_me` works: outage copy, not \$0 and not error 2.
- [ ] Failing connector does not open x.com login in a browser or ask for a Bearer token.

Made with [Cursor](https://cursor.com)

### CODE REVIEW

File: third_party/x/skills/x-api-mcp-guide/SKILL.md [hunk 1]
```diff
@@ -3,19 +3,86 @@ name: X MCP guide
...
+| Cursor Pro | $10 |
+
+If they ask how much they have **left**, quote `{credits}` (`total_balance`) — same number you already state on connect.
 
 ## On connect
 
-The first time the user connects X — or on their first X interaction in a session — send this capabilities message once. Adapt the wording to your voice, keep every line of content:
+Once tools exist and `{credits}` is cached, send this once. Adapt the wording to your voice. Keep every capability bullet.
+
+If `free_balance` **> 0**, lead with:
```
Comment (tcaldwell-x) [inline third_party/x/skills/x-api-mcp-guide/SKILL.md]: YAML says first connect, but this fires whenever free_balance > 0 and On connect is once per session (line 108). Anyone with leftover free grants hears "Congrats, you have received free X API credits to get started!" every session.

Also: public docs say total_balance is max(0, prepaid_balance + free_balance) and prepaid may be negative. So free_balance > 0 and {credits} ~$0 can both be true. Line 106 says skip congrats on $0; this line says lead with it. Make {credits} ~$0 win, and gate congrats to first connect / a new grant, not leftover free_balance.

File: third_party/x/skills/x-api-mcp-guide/SKILL.md [hunk 2]
```diff
@@ -26,50 +93,100 @@ The first time the user connects X — or on their first X interaction in a sess
...
 
-### 2. Not onboarded (403)
+If status is `connected` but tools=0, that is #2, not this.
+
+### 2. Account not ready
+
+**When (primary):** X is connected but the agent has **no tools** — the developer account is not set up or not enabled. Signatures:
 
-**When:** `client-forbidden`; `user-not-enrolled`; `client-not-enrolled`; Client Forbidden; 403 on timeline / mentions / search / bookmarks after Connect.
+- `Tool user-X-get_users_me was not found` (or any `user-X-*` / `get_users_me` / `get_usage_credits` not found)
```
Comment (tcaldwell-x) [inline third_party/x/skills/x-api-mcp-guide/SKILL.md]: Do not treat get_usage_credits not found as account-not-ready by itself. If other user-X-* tools exist, that is a missing endpoint or scope, not "create a Default Project and App."

Primary signature should stay connected + tools=0 / X namespace missing. A lone missing credits tool should be outage / could not read balance, not error 2.

**Review by tcaldwell-x — COMMENTED** [review]
Intent matches the test plan: credits before any connect copy, no purchase CTA until a real $0 / over-budget read, tools=0 is setup not a paywall, 503 is an outage, Connect-only auth. Version bump, changelog, and the get_usage_credits pricing.md line all line up. Field names match the public GET /2/usage/credits docs (total_balance / free_balance / free_grants).

Not stamping. A few spots will produce the wrong user-facing copy — inline on the two that matter, nits below.

**Nits**
- Starter table: agents usually do not know the Cursor plan. Say only quote a row if you actually know it; do not guess. No Hobby / Business / other row.
- GrokBot-specific browser wording is agent-facing in a public Cursor plugin. The quoted user line is already generic; When/Do not should say this computer / the agent browser so non-GrokBot agents still skip browser login.
- Error 2 quoted copy: "clearing out and re-installing" is awkward.

Confirm hosted MCP actually exposes get_usage_credits under the current billing.write scope. If that tool is missing while other X tools work, today error 2 signatures will tell healthy accounts to create a Project and App.

**Review by tcaldwell-x — APPROVED** [review]

**Review by tcaldwell-x — APPROVED** [review]

**Review by tcaldwell-x — APPROVED** [review]

**Review by tcaldwell-x — APPROVED** [review]

**Review by tcaldwell-x — APPROVED** [review]
